src/DeviceFirmata.cpp

```diff
@@ -0,0 +1,264 @@
+/*
+  DeviceFirmata.cpp - Firmata library
+*/
+
+#include "DeviceFirmata.h"
+#include "utility/Boards.h"
+#include <Base64.h>
```

Comment:
Client availability.
  Documentation and knowledge widely available.
  No embedded Firmata calls.  I'm trying to make sure that Luni can be used  
totally standalone, or perhaps with other comms protocols, and I didn't  
want anything that assumed the Firmata environment.


On Sun, 19 Jun 2016 23:07:42 -0700, Jeff Hoefs <notifications@github.com>  
wrote:

>
> In src/DeviceFirmata.cpp:
>> @@ -0,0 +1,264 @@
>> +/*
>> +  DeviceFirmata.cpp - Firmata library
>> +*/
>> +
>> +#include "DeviceFirmata.h"
>> +#include "utility/Boards.h"
>> +#include <Base64.h>
>
> My assumption is availability of Base64 libraries for client  
> applications, but wanted to understand if there are other  
> reasons/advantages.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.



-- 
Using Opera's mail client: http://www.opera.com/mail/